environment.yml

```diff
@@ -30,4 +30,5 @@ dependencies:
   - pip:
     - lxml==4.8.0
     - hyp3lib==1.7.0
-    - pyarrow
+    - rtree
+    - pyarrow
```

Comment:
We pip install lxml and hyp3lib due to a conda build issue (see comment in env file) but I don't see why are we installing rtree and pyarrow via pip instead of conda (they are both available via conda)

Reply:
Ok, I will move them to conda install portion